src/App.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import Title from './TitleComponent/TitleComponent'
+import './App.css'
+class App extends React.Component {
+
+    constructor(props){
```

Comment:
If you have only the state variable declaration in the constructor, so you don't have to declare the constructor. You can just declare the state = {} as a class field

Reply:
removed constructor and declared state as a class field.